jupyter/Scale Output Example.ipynb

```diff
@@ -0,0 +1,182 @@
+{
```

Comment:
@jjaco2 You probably want to review the strings in this file. They may be over explanatory.

Reply:
See comment below on this -- I think this is an appropriate level of detail. We tend to under-explain on code examples, and I like opportunities to slowly change that.